dll/shellext/zipfldr/precomp.h

```diff
@@ -0,0 +1,101 @@
+#ifndef ZIPFLDR_PRECOMP_H
+#define ZIPFLDR_PRECOMP_H
+
+#define COBJMACROS
+#define WIN32_NO_STATUS
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#define NTOS_MODE_USER
+
+#include <windef.h>
+#include <winbase.h>
+#include <shlobj.h>
+//#include <tchar.h>
+#include <atlbase.h>
+#include <atlcom.h>
+#include <atlcoll.h>
+#include <atlstr.h>
+#include <atlwin.h>
+#include <shlwapi.h>
+#include <shellapi.h>
+#include <strsafe.h>
+#include <rosdlgs.h>
+#include <assert.h>
+
+EXTERN_C
+ULONG DbgPrint(PCSTR Format,...);
```

Comment:
We should think about moving this somewhere shared...

Reply:
Isn't it already in debug.h ?

Reply:
`boot/freeldr/freeldr/include/debug.h`
That doesn't really help outside freeldr ;)

Reply:
Ah, I thought it was also in [sdk/include/reactos/debug.h](https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/debug.h;hb=851a554b5922af7ac1a21b12fd788ebb051641b8#l27).